tx_pool/src/pool.rs

```diff
@@ -239,6 +238,16 @@ impl Pool {
         tx_list
     }
 
```

Comment:
Not sure that could we add `#cfg(test)` here ?

Reply:
No, this is a pub function.